### ISSUE TITLE
PWA Could Not Extract Zip

### ISSUE THREAD

**Issue by wppronto** [uncatcrea/wp-appkit#339]
Hi,

Whenever we try to install the PWA from the plugin, we get "Could not extract zip"... error. PHP configuration seems right (running 5.6), and file permissions don't seem to prohibit the operations.

Any idea? :-)

Thanks,

Mark

**Comment by lpointet** [comment]
Hi Mark,

It could be related to the following WordPress regression: https://core.trac.wordpress.org/ticket/42026
This issue will be addressed in the 4.9 release (see https://core.trac.wordpress.org/ticket/42016).

In the meantime, you can try commenting the lines where the following code appear in WP core:
```
if ( 0 !== validate_file( $info['name'] ) ) {
    return new WP_Error( 'invalid_file_ziparchive', __( 'Could not extract file from archive.' ), $info['name'] );
}
```

Let us know if this work.

Best regards,
Lionel

**Comment by blupu** [comment]
Not active - closing.
